manifests/i/Inkscape/Inkscape/0.92.5/Inkscape.Inkscape.installer.yaml

```
@@ -0,0 +1,35 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=QUSU.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Inkscape.Inkscape
+PackageVersion: 0.92.5
```

```suggestion
PackageVersion: 0.92.5.0
```
This should match what is written to control panel